### PR TITLE
Newsletter-253-Translate-into-French

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Comment by Copinmalin** [comment]
@bitschmidty  hello, can you check if it's ok for this newsletter please ?

File: _posts/fr/newsletters/2023-05-31-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  avec [taproot][topic taproot] et [tapscript][topic tapscript] (respectivement, BIPs [341][bip341] et [342][bip342]) utilisent des
+  messages de 32 octets.
+
+    Les ajouts décrivent comment utiliser efficacement des messages de longueur arbitraire, recommandent l'utilisation d'un préfixe
+    de balise haché et fournissent des recommandations pour accroître la sécurité lors de l'utilisation de la même clé dans
+    différents domaines (tels que la signature de transactions ou la signature de messages en texte clair).
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="27469,27626,25796,2668,2666,4689,1446" %}
+[policy series]: /en/blog/waiting-for-confirmation/
```
[the remark this answers is not in the document]
Comment (Copinmalin) [reply _posts/fr/newsletters/2023-05-31-newsletter.md:249]: OK, what I need to do ?

[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/fr/newsletters/2023-05-31-newsletter.md:249]: Create a fr version of /en/blog/waiting-for-confirmation/ and link accordingly. 

Checkout the ja equivalent PR: https://github.com/bitcoinops/bitcoinops.github.io/pull/1134/files#diff-b033551ad1b8fa5fb8d47b44306ad3e1b09a96857d19835506eef5f519843bbc

[the remark this answers is not in the document]
Comment (Copinmalin) [reply _posts/fr/newsletters/2023-05-31-newsletter.md:249]: @bitschmidty  : Like this ?

[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/fr/newsletters/2023-05-31-newsletter.md:249]: This link should be the /fr/ and not /en/ one

File: _posts/fr/newsletters/2023-05-31-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    Le message de M. Jager contenait un lien vers une [vidéo][jager video] où il faisait une démonstration de la fonction.
+    À l'heure où nous écrivons ces lignes, son message n'a reçu que quelques réponses, toutes positives.
+
+## Attente de la confirmation #3 : Enchères pour l'achat d'un bloc d'espace
+
+_Une [série][policy series] hebdomadaire limitée sur le relais de transaction, l'inclusion dans le mempool et la sélection
+des transactions minières---y compris pourquoi Bitcoin Core a une politique plus restrictive que celle permise par le
+consensus et comment les portefeuilles peuvent utiliser cette politique de la manière la plus efficace._
+
+{% include specials/policy/en/03-bidding-for-block-space.md %}
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-05-31-newsletter.md:143]:
```suggestion
{% include specials/policy/fr/03-encheres-pour-l-achat-d-un-bloc-d-espace.md %}
```

File: _posts/fr/newsletters/2023-05-31-newsletter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/fr/newsletters/2023-05-31-newsletter.md:249]: /fr/blog/attente-de-la-confirmation/ I believe

**Comment by Copinmalin** [comment]
@bitschmidty : help needed
image

File: _includes/specials/policy/fr/03-encheres-pour-l-achat-d-un-bloc-d-espace.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+de la transaction. Comme les transactions se concurrencent par taux [frais/poids], une transaction plus légère nécessite des frais
+moins élevés pour atteindre le même taux.
+
+Certains portefeuilles, comme le portefeuille Bitcoin Core, essaient de combiner les entrées de manière à éviter d'avoir besoin de
+changer les sorties. Éviter le changement permet d'économiser le poids d'une sortie maintenant, mais aussi le coût futur de la
+dépense de la sortie de changement plus tard. Malheureusement, de telles combinaisons d'entrées ne seront que rarement disponibles,
+à moins que le portefeuille ne dispose d'un grand pool d'UTXO avec une grande variété de montants.
+
+Les types de sortie modernes sont plus efficaces en termes d'espace de blocs que les types de sortie plus anciens. Par exemple,
+dépenser une entrée P2TR coûte moins de 2/5e du poids d'une entrée P2PKH (essayez-le avec notre [calculateur de taille de transaction
```
Comment (bitschmidty) [inline _includes/specials/policy/fr/03-encheres-pour-l-achat-d-un-bloc-d-espace.md]: Took me way too long to find this as the issue

```suggestion
dépenser une entrée P2TR coûte moins de 2/5e du poids d'une entrée P2PKH (essayez-le avec notre [calculateur de taille de transaction]
```

Pushed a commit to fix this and also removed what appears to be an extra "(" on the next line

**Comment by bitschmidty** [comment]
Also in the fixup commit for the broken link, I renamed the series blog post to the en version. This is both consistent with the cs/ja translations but also ensures the proper translation links at the top of the blog page.

**Comment by bitschmidty** [comment]
There was also a broken anchor link as well in the fixup

**Comment by bitschmidty** [comment]
@Copinmalin can you take a look/review?

**Comment by Copinmalin** [comment]
Many thanks @bitschmidty , sorry for extrawork to help me !

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin !

**Pull request merged**